well-known-bots.json

```diff
@@ -173,6 +173,7 @@
   {
     "id": "google-crawler-other",
     "categories": [
+      "ai",
```

Comment:
Is GoogleOther AI or is just "unknown"? Aren't the AI related ones only [Google-CloudVertexBot](https://developers.google.com/search/docs/crawling-indexing/google-common-crawlers#google-extended) and [Google-Extended](https://developers.google.com/search/docs/crawling-indexing/google-common-crawlers#google-extended)?

Reply:
> Crawling preferences addressed to the GoogleOther user agent don't affect any specific product. GoogleOther is the generic crawler that may be used by various product teams for fetching publicly accessible content from sites. For example, it may be used for one-off crawls for internal research and development. It has no effect on Google Search or other products. 

However, DarkVisitors and ai.robots.txt have marked it as an "AI" bot. Google-CloudVertexBot is explicitly a crawler for the CloudVertex product/model, but Google-Extended is not an actual user-agent, but an opt-out flag in the robots.txt file for ALL google products:

> Google-Extended doesn't have a separate HTTP request user agent string. Crawling is done with existing Google user agent strings; the robots.txt user-agent token is used in a control capacity. 

> Google-Extended is a standalone product token that web publishers can use to manage whether their sites help improve Gemini Apps and Vertex AI generative APIs, including future generations of models that power those products. Google-Extended does not impact a site's inclusion or ranking in Google Search.

Reply:
Yeah, we'll have to add a note to the docs about not being able to block GoogleExtended and that they should add it to their robots.txt.

Re: google-crawler-other do you think we have enough evidence to confidently list it as AI?

Reply:
> Re: google-crawler-other do you think we have enough evidence to confidently list it as AI?

I've been trusting ai.robots.txt but dark visitors says it is [not currently known](https://darkvisitors.com/agents/googleother) to be AI so I'm fine leaving it out. I suppose we remove the AI category from applebot, too, since they really on the "extended" in the robots.txt?

Reply:
```suggestion
```